build.gradle

```diff
@@ -73,16 +74,30 @@ subprojects {
         mavenCentral()
         mavenLocal()
     }
-
-    checkstyle {
-        toolVersion = '8.12'
-        configFile = rootProject.file('config/checkstyle/checkstyle.xml')
+    
+    spotless {
+        java {
+            target project.fileTree(project.rootDir) {
+                include '**/*.java'
+                exclude '**/package-info.java'
+            }
+            importOrder 'cern', 'com', 'io', 'org', 'reactor', 'java', 'javax', 'static *'
```

Comment:
Basically, alphabetical except for `java*`, with statics last.

Reply:
Yes, I understand that. I'm not sure that there is other way to specify import order with having `java*` last exept specifying all domains that should be before explicitly